scripts/pre_commit_linter.py

```diff
@@ -180,6 +186,20 @@
 _MESSAGE_TYPE_FAILED = 'FAILED'
 
 
+def _is_filename_excluded_for_bad_patterns_check(pattern, filename):
+    """Checks if file is excluded from the bad patterns check
+
+    Args:
+    - pattern: The pattern to be checked against.
```

Comment:
Changing the other occurrences would be great, if you're up for it. The style I pointed you at is the style we use in the codebase in general -- see e.g. #2465.